packages/app/src/DashboardFilters.tsx

```diff
@@ -44,6 +44,8 @@ const DashboardFilterSelect = ({
         select: '',
       },
       keys: [filter.expression],
+      disableRowLimit: true,
+      limit: 10000,
```

Comment:
Increasing this to match other filters (on the search page and the existing servicename filter on the services dashboard)